ga4gh/server/serverconfig.py

```diff
@@ -93,6 +93,20 @@ class GoogleOidcConfig(ProductionConfig):
     OIDC_CLIENT_SECRET = "XXX"
 
 
+class Auth0Config(DevelopmentConfig):
+    """
+    Auth0 configuration
+    """
+    AUTH0_ENABLED = True
```

Comment:
This is a configuration setting for testing. To use it you provide `-c Auth0Config` at the command line.